controllers/cluster_controller_test.go

```diff
@@ -367,6 +368,7 @@ func TestClusterReconcilerReconcileConditions(t *testing.T) {
 			log := testr.New(t)
 			logCtx := ctrl.LoggerInto(ctx, log)
 
+			mockPkgs.EXPECT().UpdateSecrets(logCtx, testClient, sameName(config.Cluster))
```

Comment:
maybe we should think about adding a helper to group all expectations? most tests apparently don't care about the expectations, just about the result